include/mbedtls/memory_buffer_alloc.h

```diff
@@ -90,6 +90,19 @@ void mbedtls_memory_buffer_set_verify( int verify );
  */
 void mbedtls_memory_buffer_alloc_status( void );
 
+/**
+ * \brief   Get the number of alloc/free so far.
+ *
+ * \param alloc_count   Number of allocations.
+ * \param free_count    Number of frees.
+ */
+void mbedtls_memory_buffer_alloc_count_get( size_t *alloc_count, size_t *free_count );
+
+/**
+ * \brief   Reset alloc/free counters.
+ */
+void mbedtls_memory_buffer_alloc_count_reset( void );
```

Comment:
Not a blocker because this is just debugging info, but this makes the final report of allocation counts wrong. I'd prefer to put the responsibility on the caller: instead of doing
```
count_reset()
do_stuff()
print(count_get())
```
the benchmark program could do
```
old = count_get()
do_stuff()
new = count_get()
print(new - old)
```